pkg/types/installconfig.go

```diff
@@ -85,6 +90,13 @@ type InstallConfig struct {
 	// +optional
 	AdditionalTrustBundle string `json:"additionalTrustBundle,omitempty"`
 
+	// AdditionalTrustBundlePolicy defines conditional policy to add AdditionalTrustBundle
```

Comment:
Can/should we move this to a boolean value? There are only two possible values that you have defined "default" which is technically not an additional trust bundle or "always". If you think that more values are possible then a string would definitely be correct but right not it seems like its True or False, right?

Reply:
Thanks for the review, @barbacbd . The boolean does present a good option if the field is bound with strict yes/no meaning. The additionalTrustBundle is not strictly bound to only proxy, I think, it is better to have usage policy represented as explicit string for extendability and to describe well on the usage of the additionalTrustBundle.

Reply:
> Can/should we move this to a boolean value?

I agree with the question, but I have learned we try to avoid booleans in the API: c.f. https://github.com/openshift/api/pull/1057#discussion_r750562936